playbooks/provider/lago/config.yml

```diff
@@ -52,7 +52,7 @@
     - name: Start Lago env
       command: lago --workdir "{{ prefix }}" start
 
-- hosts: all
+- hosts: nodes
```

Comment:
Should be all the VMs that lago creates, you can keep it as it is.
In ansible 2.5.3, the `all` pattern doesn't include the local host:

```
 > ansible -i /dev/null --list-hosts "all"
 [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

  hosts (0):
```

Reply:
no sweat... reverting that bit